index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en" ng-app="ToDoList">
+
+<head>
+  <meta charset="UTF-8">
+  <title>ToDoList</title>
+  <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.css">
+  <link rel="stylesheet" href="public/css/custom.css">
+  <script src="bower_components/jquery/dist/jquery.js"></script>
+  <script src="bower_components/angular/angular.js"></script>
+  <script src="bower_components/angular-resource/angular-resource.js"></script>
+  <script src="js/toDoListController.js"></script>
+</head>
+
+<body ng-controller="MainController">
```

Comment:
consider pros and cons of "controller as" syntax? (http://toddmotto.com/digging-into-angulars-controller-as-syntax/)